src/main/java/edu/ucdavis/dss/ipa/entities/InstructorCost.java

```diff
@@ -89,6 +101,16 @@ public long getInstructorIdentification() {
         }
     }
 
+    @JsonProperty("instructorTypeId")
+    @Transient
+    public long getInstructorTypeIdentification() {
+        if(instructorType != null) {
+            return instructorType.getId();
+        } else {
+            return 0;
```

Comment:
Is 0 not a valid instructor type ID? Why not make it 'Long' instead of 'long' and return null?